pkg/controller/job/job_controller_test.go

```diff
@@ -2593,19 +2588,18 @@ func TestSyncJobWithJobPodFailurePolicy(t *testing.T) {
 	}
 
 	testCases := map[string]struct {
-		enableJobPodFailurePolicy     bool
-		enablePodDisruptionConditions bool
-		enableJobPodReplacementPolicy bool
-		job                           batch.Job
-		pods                          []v1.Pod
-		wantConditions                *[]batch.JobCondition
-		wantStatusFailed              int32
-		wantStatusActive              int32
-		wantStatusSucceeded           int32
-		wantStatusTerminating         *int32
+		enablePodDisruptionConditions      bool
+		TestSyncJobWithJobPodFailurePolicy bool
```

Comment:
This field doesn't seem to be used anywhere

Reply:
Right, not sure it got there, but removed now :)